plan_delegator/src/plan_delegator.cpp

```diff
@@ -607,7 +607,7 @@ namespace plan_delegator
 
         // Track the index of the starting maneuver in the maneuver plan that this trajectory plan service request is for
         uint16_t current_maneuver_index = 0;
-
+        int count_expired_maneuvers = 0;
```

Comment:
We skip manuevers that are "expired" and "passed", so maybe we can make this more general by renaming to `count_skipped_maneuvers`?